--- networkx-3.5/pyproject.toml	2025-05-29 23:34:32.000000000 +1200
+++ networkx-3.5-b/pyproject.toml	2025-10-17 17:47:07.119245684 +1300
@@ -127,7 +127,6 @@
     'networkx.linalg',
     'networkx.readwrite',
     'networkx.readwrite.json_graph',
-    'networkx.tests',
     'networkx.utils',
 ]
 platforms = [
@@ -141,43 +140,13 @@
 attr = 'networkx.__version__'
 
 [tool.setuptools.package-data]
-networkx = ['tests/*.py']
-"networkx.algorithms" = ['tests/*.py']
-"networkx.algorithms.assortativity" = ['tests/*.py']
-"networkx.algorithms.bipartite" = ['tests/*.py']
-"networkx.algorithms.centrality" = ['tests/*.py']
-"networkx.algorithms.community" = ['tests/*.py']
-"networkx.algorithms.components" = ['tests/*.py']
-"networkx.algorithms.connectivity" = ['tests/*.py']
-"networkx.algorithms.coloring" = ['tests/*.py']
-"networkx.algorithms.minors" = ['tests/*.py']
-"networkx.algorithms.flow" = [
-    'tests/*.py',
-    'tests/*.bz2',
-]
-"networkx.algorithms.isomorphism" = [
-    'tests/*.py',
-    'tests/*.*99',
-]
-"networkx.algorithms.link_analysis" = ['tests/*.py']
-"networkx.algorithms.approximation" = ['tests/*.py']
-"networkx.algorithms.operators" = ['tests/*.py']
-"networkx.algorithms.shortest_paths" = ['tests/*.py']
-"networkx.algorithms.traversal" = ['tests/*.py']
-"networkx.algorithms.tree" = ['tests/*.py']
-"networkx.classes" = ['tests/*.py']
 "networkx.generators" = [
-    'tests/*.py',
     'atlas.dat.gz',
 ]
 "networkx.drawing" = [
     'tests/*.py',
     'tests/baseline/*png',
 ]
-"networkx.linalg" = ['tests/*.py']
-"networkx.readwrite" = ['tests/*.py']
-"networkx.readwrite.json_graph" = ['tests/*.py']
-"networkx.utils" = ['tests/*.py']
 
 [tool.changelist]
 ignored_user_logins = ["dependabot[bot]", "pre-commit-ci[bot]", "web-flow"]
